backend/src/api/board.rs

```diff
@@ -0,0 +1,20 @@
+use actix_web::{ post, web, HttpResponse, Responder };
```

Comment:
nitpick, `cargo fmt` as your indentation is inconsistent :)

Reply:
so it is! added a commit w/ `cargo fmt` changes